docs/components/concepts/job-workers.md

```diff
@@ -72,6 +72,11 @@ This could be useful when a job worker communicates with an external system. If
 This will result in an incident when the retries run out. Using the `retry back off` will delay the retry. This allows the external system some time to recover.
 If no `retry back off` the job is immediately retried.
 
+When `Completing or failing jobs` with variables, the variables are merged into the process of the job's associated task.
+
+- If `Completing a job` the variables are propagated from the scope of the task to its higher scopes.
+- If `Failing a job` the variables are only created in the local scope of the task.
```

Comment:
🔧 I'm not a native English speaker, but I think it should be:

```suggestion
- When `Completing a job` the variables are propagated from the scope of the task to its higher scopes.
- When `Failing a job` the variables are only created in the local scope of the task.
```

Reply:
Good catch 👍